main.go

```diff
@@ -270,6 +272,15 @@ func main() {
 		},
 	}
 
+	cmdVersion := &cobra.Command{
+		Use:   "version",
+		Short: "Print knest version.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Printf("knest version: %#v, virtink version: %#v, cluster-api-provider-virtink version: %#v\n", version, VirtinkVersion, VirtinkProviderVersion)
```

Comment:
Virtink should be capitalized. And should we print in JSON format?

Reply:
add `--output` flag to specify output format.